### PR TITLE
Added the docs for WebServerBundle

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony-docs/issues/7325
| License       | MIT
| Doc PR        | -

This initial doc is needed to add the WebServerBundle to this page: https://symfony.com/doc/bundles/

### CODE REVIEW

**Comment by stof** [comment]
We never put documentation in the main Symfony repo. This should go in symfony-docs

File: src/Symfony/Bundle/WebServerBundle/Resources/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+``--format``
+    The line format applied to every log message before displaying it (e.g.
+    ``ConsoleFormatter::SIMPLE_FORMAT``).
+
+``--date-format``
+    The format applied to the date of the log message (e.g. ``ConsoleFormatter::SIMPLE_DATE``).
+
+``--filter``
+    Any `ExpressionLanguage`_ compatible expression used to filter the log
+    messages before displaying them (e.g. ``"level > 200 or channel in [\'app\', \'doctrine\']"``).
```
Comment (stof) [inline src/Symfony/Bundle/WebServerBundle/Resources/doc/index.rst]: you should document the available variables

**Comment by javiereguiluz** [comment]
If we don't document this in the bundle's repository, then we won't be able to display it on https://symfony.com/doc/bundles/

Besides, other bundles already have short docs in this repo: https://github.com/symfony/monolog-bundle/blob/master/Resources/doc/index.rst and https://github.com/symfony/swiftmailer-bundle/blob/master/Resources/doc/index.rst

**Comment by weaverryan** [comment]
Ah, so the difference between this bundle and *those* bundles is that they are in separate repositories. But, since we're moving towards not including `symfony/symfony`, it does indeed make more sense to begin documenting the individual bundles as if they were separated.

So, 👍 in theory, but I kinda doubt that Fabien will want big docs like this living in this repo. Would it be a better idea (for bundles that are actually *inside* symfony/symfony) to add a `bundles/` directory in `symfony/symfony-docs` and update `symfony.com` to be able to pull from that?

**Comment by stof** [comment]
I like the idea of @weaverryan

**Comment by fabpot** [comment]
I've just pushed an alternative in #22607. I think it's better to not duplicate documentation. Commands should be documented so that help is available on the CLI. I've also reused the description of the component in the README. That should be enough I think.

**Comment by robfrawley** [comment]
@fabpot Basic usage and installation should be documented on the Symfony website under the `bundles/` path, regardless of the completeness of the CLI documentation, IMHO.

**Comment by fabpot** [comment]
@robfrawley Installation is a no-brainer with Flex. And there is no documentation except the CLI commands as the bundle is just about that. I don't see the point.

**Pull request closed without merging**